packages/sw-routing/src/lib/express-route.js

```diff
@@ -66,8 +74,8 @@ class ExpressRoute extends Route {
    *        HTTP method. Defaults to `'GET'` if not specified.
    */
   constructor({path, handler, method}) {
-    if (path.substring(0, 1) !== '/') {
-      throw ErrorFactory.createError('express-route-requires-absolute-path');
+    if (!(path.startsWith('/') || path.startsWith('https://'))) {
```

Comment:
I'll open it up to `http` if we want to support that use case.

I do think that this check is needed, yes. If they don't pass in something that starts with `'/'` or `http` then it's not a valid `path`, so shouldn't we tell them that? The alternative would be to allow them to create an `ExpressRoute` that will never match anything, and that doesn't seem preferable.